Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift

```diff
@@ -19,6 +19,14 @@ extension ChatViewController: UIImagePickerControllerDelegate, UINavigationContr
             alert.addAction(UIAlertAction(title: localized("chat.upload.take_photo"), style: .default, handler: { (_) in
                 self.openCamera()
             }))
+
+            alert.addAction(UIAlertAction(title: localized("chat.upload.shoot_video"), style: .default, handler: { (_) in
+                AVAudioSession.sharedInstance().requestRecordPermission { granted in
+                    if granted {
+                        self.openCamera(video: true)
+                    }
```

Comment:
If not, let's show an error message?

Reply:
Looks like camera handles that for us, so I'm removing this part :)